README.md

```diff
@@ -181,12 +183,40 @@ You can now use curl, for example:
 curl -H "Authorization: Bearer $TOKEN" http://localhost:9092/api/v1/query\?query\=net_conntrack_dialer_conn_attempted_total
 ```
 
+#### Proxy to Amazon Managed Service for Prometheus
+
+All requests to an AWS managed prometheus service need a signature in the `Authorization` header,
+which is calculated based on the request URL, headers, and body.
+Since the proxy modifies the request on the fly, any existing signature will
+be invalidated.
+This is why prometheus-multi-tenant-proxy incorporates AWS signature v4.
+
+To enable AWS signature, use either the `--aws` flag or set the environment variable
+`PROM_PROXY_USE_AWS=true`.
```

Comment:
Interesting!
It would be nice having other option as environment variables.
If so, we should use the `PROM_PROXY_` prefix :)

Good tip!